sdk/iot/iot-model-parser/CHANGELOG.md

```diff
@@ -0,0 +1,47 @@
+# Release History
+
+## 1.0.10-beta.1 (2020-11-16)
+
+- Test Release Pipeline
+
+## 1.0.9-beta.13 (2020-10-13)
+
+- Test Release Pipeline
+
+## 1.0.9-beta.3 (2020-09-05)
```

Comment:
any idea what is going on here? did we actually release this package before or this template was copied from somewhere else by mistake?

Reply:
copied by mistake. updating.